pulp_smash/config.py

```diff
@@ -489,6 +490,13 @@ def get_base_url(self, pulp_host=None, role="api"):
             pass
         return urlunsplit((scheme, netloc, "", "", ""))
 
+    def get_fixtures_url(self):
+        """Return fixtures URL."""
+        local_fixture = self.custom.get("local_fixture")
+        if local_fixture:
+            return urljoin(self.get_base_url(role="api"), "/static/fixtures/")
```

Comment:
Can you have this default to `https://repos.fedorapeople.org/pulp/pulp/fixtures/`?